equinity-historical-data/src/main/java/uk/gov/justice/laa/crime/equinity/historicaldata/model/Crm14ChargesModel.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.justice.laa.crime.equinity.historicaldata.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.Date;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class Crm14ChargesModel {
+    @JsonProperty("Offence_date_1")
+    public Date offence_date_1;
+    @JsonProperty("Offence_date_2")
+    public Date offence_date_2;
+    @JsonProperty("Offence_date_3")
+    public Date offence_date_3;
+    @JsonProperty("Offence_when")
+    public String offence_when;
```

Comment:
Date names are confusing.  What's the difference between 1, 2, 3 and when ? It would be good to have a more self explanatory names

Reply:
If offence_when  is Between
Then we have Offence_date_2 & Offence_date_3